code/ui/generalhud/inventorywrapper/effects/Effects.scss

```diff
@@ -1,47 +1,44 @@
 Effects {
     display: flex;
-    justify-content: flex-end;
-    font-size: 12px;
-    font-family: Poppins;
-    margin-bottom: 10px;
+    align-items: flex-end;
+    flex-direction: column-reverse;
+    margin-bottom: 25px;
+}
 
-    Effect {
-        border: 1px solid #031421;
-        background-color: rgba(0, 0, 0, 0.7);
-        border-radius: 4px;
-        width: 62px;
-        height: 62px;
-        margin-left: 10px;
-        margin-top: 10px;
+Effect {
```

Comment:
Why isn't the effect in the effects class anymore?

Reply:
I made everything separate classes. Too many things had nested classes and they were annoying to work with, resulting in stuff like `Parent.Parent.Parent`. I figured everything should be in the same scope and tried to maintain consistency even if I didn't absolutely need to.